Development versions are only for testing (#3738)
chef/chef-web-docs: content/versions.md

## Supported Commercial Distributions

{{< note >}}

> Development releases of any product are not supported for any usage other than testing. A stable build may be produced from the results of testing with a development build, but support can be offered for stable versions only.

Reviewer: ```suggestion
We do not support development releases of our products for any usage other than testing. You may be able to produce a stable build from a development release, but we only offer support for stable releases.
```
